File: DofusSpa/src/app/_models/idol-recipe.ts
import { Idol } from "./idol";

export interface IdolRecipe{
    id : number,
    ankamaID : number,
    name : string,
    url : string,
    imgUrl : string,
    quantity : number,
    level : number,
    type: string,
    idolID : number,
    idol : Idol
}

File: DofusApi/Models/IdolRecipe.cs
namespace DofusApi.Models
{
    public class IdolRecipe
    {
        public int ID { get; set; }
        public int AnkamaID { get; set; }
        public string Name { get; set; }
        public string Url { get; set; }
        public string ImgUrl { get; set; }
        public int Quantity { get; set; }
        public int Level { get; set; }
        public string Type { get; set; }
        public int IdolID { get; set; }
        public Idol Idol { get; set; }
    }
}